app/src/routes/configure.js

```diff
@@ -1,72 +1,187 @@
 import Selector from "../components/selector";
-import Slider from "../components/slider";
 import HelpButton from "../components/helpButton";
 import TextForm from "../components/textForm";
+import { useEffect, useState } from "react";
+import { collection, onSnapshot } from "firebase/firestore";
+import { firebaseDb } from "../firebase";
 
-function StyledLine({ title, inputValue, inputUnits, inputComponent, helpButtonComponent}) {
+function StyledLine({ title, inputComponent, helpButtonComponent}) {
   return (
-    <div  className="border-b border-gray-300 pb-2">
-      <div className="p-2 grid grid-cols-4 gap-2">
-        <h4 className="text-lg font-semibold">{title} </h4>
-        <div className="flex items-center">
-          <span className="mr-1">{inputValue} {inputUnits}</span>
-        </div>
-        <div className="flex justify-center items-center">
-          {inputComponent}
-        </div>
-        {helpButtonComponent}
+    <div>
+      <div className="flex py-6">
+          <div className="w-60 flex pl-3">
+              <div className="mx-2">
+                <div className="text-lg font-semibold">{title} </div>
+              </div>
+              <div className="">
+                  {helpButtonComponent}
+              </div>
+          </div>
+          <div className="">
+              {inputComponent}
+          </div>
       </div>
-      
     </div>
+    
   );
 }
 
-function EmailForm({onChange, email}) {
-  return(
-    <TextForm type="email" placeholder="Enter your email" onChange={onChange} value={email}/>
-  )
+function TextArea({value, onChange}) {
+  return (
+    <div className="">
+      <textarea class="px-2 py-1 bg-gray-50 border border-gray-300 text-gray-900 rounded-md focus:ring-blue-500 focus:border-blue-500 block" 
+                value={value} onChange={onChange} rows="4" cols="50"/>
+    </div>
+  );
 }
 
-function NameForm({onChange, name}) {
-  return(
-    <TextForm type="text" placeholder="Enter your name" onChange={onChange} value={name}/>
+function MaterialSelector({init, materials, changeMaterial}) {
+  const [showOptions, setShowOptions] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(init);
+
+  const handleItemClick = (item) => {
+    setSelectedItem(item);
+    changeMaterial(item.Type)
+  };
+
+  return (
+    <div>
+      <button onClick={() => setShowOptions(true)}>
+        <div className="flex">
+          {showOptions ? 
```

Comment:
I'm thinking this code would look cleaner if it was separated into a renderMaterials function, but that might just be my personal preference